schemas/bridging/common.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://fdc3.finos.org/schemas/next/api/common.schema.json",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/common.schema.json",
```

Comment:
This error broke local file resolution during code generation, causing quicktype to retrieve the currently published version on the web (which has the same issue).